server/events/instrumented_pull_closed_executor.go

```diff
@@ -18,9 +18,15 @@ type InstrumentedPullClosedExecutor struct {
 func NewInstrumentedPullClosedExecutor(
 	scope tally.Scope, log logging.SimpleLogging, cleaner PullCleaner,
 ) PullCleaner {
+	scope = scope.SubScope("pullclosed_cleanup")
+
+	for _, m := range []string{metrics.ExecutionSuccessMetric, metrics.ExecutionErrorMetric} {
```

Comment:
How come `metrics.ExecutionFailureMetric` isn't listed here like in `server/events/instrumented_project_command_runner.go`

Reply:
sounds reasonable